src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    static final int DATE = 0;
+    static final int NAME = 1;
+    static final int WORKING_HOURS = 2;
+    static final int WAGE_PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, DATE_FORMATTER);
+        LocalDate tillDate = LocalDate.parse(dateTo, DATE_FORMATTER);
+        StringBuilder builder = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder reportBuilder = new StringBuilder();
```

lets make variable names more informative